cmd/horcrux/cmd/root.go

```diff
@@ -30,13 +30,14 @@ func Execute() {
 
 func init() {
 	cobra.OnInitialize(initConfig)
-	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.horcrux/config.yaml)")
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "home", "", "directory for config and data (default is $HOME/.horcrux/)")
 }
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
 	cfg := Config{}
 	if cfgFile != "" {
+		cfgFile = path.Join(cfgFile, "config.yaml")
```

Comment:
If you make the above change, then this will need to change too.

Reply:
I've tweaked things now where `homeDir` will point to the actual directory but this part will remain `cfgFile` since it actually points to the `config.yaml` file